db/data/20201008050453_migrate_permissions_to_accesses.rb

```diff
@@ -0,0 +1,26 @@
+class MigratePermissionsToAccesses < ActiveRecord::Migration[5.2]
+  require "tasks/scripts/create_accesses_from_permissions"
+
+  def up
+    # This is a dev-env only migration
+    unless Rails.env.development?
+      return
+    end
+
+    Organization.all.each do |organization|
+      CreateAccessesFromPermissions.do(organization: organization, dryrun: false)
```

Comment:
I assume we can delete this task + old tables etc. in a single PR separately.

Reply:
Yup 👍